src/components/expandableSection/index.tsx

```diff
@@ -17,24 +17,29 @@ export type ExpandableSectionProps = {
    * should the expandableSection be expanded
    */
   expanded?: boolean;
+  /**
+   * should the expandableSection open above the sectionHeader
+   */
+  top?: boolean;
```

Comment:
Let's add an example demonstrating it in the example screen..
you can add a toggle that switch between the two behaviors..